orchestrator/workflow.py

```diff
@@ -178,12 +180,18 @@ def form_generator(state: State) -> FormGenerator:
     return form_generator
 
 
+def allow(_: OIDCUserModel | None = None) -> bool:
+    """Default function to return True in absence of user-defined authorize function."""
+    return True
+
+
 def make_workflow(
     f: Callable,
     description: str,
     initial_input_form: InputStepFunc | None,
     target: Target,
     steps: StepList,
+    authorize_callback: Callable[[OIDCUserModel | None], bool] | None = None,
```

Comment:
Could this work too?

```suggestion
    authorize_callback: Callable[[OIDCUserModel | None], bool] = allow,
```

Reply:
Unfortunately linting tests are failing. I think in order to keep the user-facing interface with a default of `None`, the signature has to be `| None = None` to make mypy happy